Formula/gpg-tui.rb

```diff
@@ -1,8 +1,8 @@
 class GpgTui < Formula
   desc "Manage your GnuPG keys with ease! 🔐"
   homepage "https://github.com/orhun/gpg-tui"
-  url "https://github.com/orhun/gpg-tui/archive/refs/tags/v0.7.4.tar.gz"
-  sha256 "12c61fac67f6f9b90c9f92e5ae754652b024ea3f85b99a2ac359f44356c49467"
+  url "https://github.com/orhun/gpg-tui/archive/refs/tags/v0.8.0.tar.gz"
```

Comment:
Might be chance to change to more common style:
```suggestion
  url "https://github.com/orhun/gpg-tui/archive/v0.8.0.tar.gz"
```
There are only ~100 formulae that include `refs/tags` vs ~2000 that use shorter URL

Reply:
yeah, I was thinking about that as well.